File: ExLab/Java/Lab2/Ex2.java
/**
 *
 * autor: Pedro Henrique Nardoni Ruiz
 * 
 * data: 9/3/2018
 * 
 * código matéria: CC3642
 * 
 * propósito: criar um programa com uma classe Date e exibir a data no formato mês/dia/ano
 */
 
package Ex2; 

class Date{   //cria classe date
    int mes;  //atributos da classe
    int dia;
    int ano;
    
    public void setMes(int novoMes){  //funçao set com verificação de entrada
        if(novoMes < 1)
            mes = 1;
        else if(novoMes > 12)
            mes = 12;
        else
            mes = novoMes;
    }
    
    public void setDia(int novoDia){ //função set com verificação de entrada
        if(novoDia < 1)
            dia = 1;
        else if(novoDia > 31)
            dia = 31;
        else
            dia = novoDia;
    }
    
    public void setAno(int novoAno){ //função set com verificação de entrada
        if(novoAno < 0)
            ano = 0;
        else
            ano = novoAno;
    }
    
    public int getMes(){  //função get
        return mes;
    }
    
    public int getDia(){  //função get
        return dia;
    }
    
    public int getAno(){  //função get
        return ano;
    }
    
    public void displayDate(){ 
        System.out.println(getMes() + "/" + getDia() + "/" + getAno());
    } //função que imprime data no formato mes/dia/ano
}



public class Ex2Java {
    public static void main(String[] args){ //classe main roda o programa
        Date d = new Date(); // instancia objeto
        
        d.setMes(3); //da o mes ao objeto
        d.setDia(2); //da o dia ao objeto
        d.setAno(2018); //da o ano ao objeto
        d.displayDate() // imprime a data escolhida 
    }
    
}

File: ExLab/Cpp/Lab2/Ex2.cpp


/*
*autor: Pedro Henrique Nardoni Ruiz
*
*data: 9/3/2018
*
*codigo da materia: CC3642
*
*explicação: criar um programa com uma classe Date e exibir a data no formato mês/dia/ano
*
*/
#include <iostream>
using namespace std;




class Date{ //cria classe date
	public: //metodos da classe
		void setMes(int novoMes){ //função set mes com verificaçao de entrada
			if(novoMes < 1)
        mes = 1;
      else if(novoMes > 12)
        mes = 12;
      else
        mes = novoMes;
		}
		
		void setDia(int novoDia){ //função set dia com verificaçao de entrada
			if(novoDia < 1)
        dia = 1;
      else if(novoDia > 31)
        dia = 31;
      else
        dia = novoDia;
		}
		
		void setAno(int novoAno){ //função set ano com verificaçao de entrada
			if(novoAno < 0)
        ano = 0;
      else
        ano = novoAno;
		}
		
		int getMes(){ //função get mes
			return mes;
		}
		
		int getDia(){ //função get dia
			return dia;
		}
		
		int getAno(){ //função get ano
			return ano;
		}
		
		void displayDate(){ //função que imprime a data no formato mes/dia/ano
			cout<<getMes();
			cout<<"/";
			cout<<getDia();
			cout<<"/";
			cout<<getAno()<<endl;
		}
	
	private: //atributos da classe
		int dia;
		int mes;
		int ano;
};
int main() {
  Date d; //instanciando objeto
  
  d.setMes(3); //atribuindo o mes
  d.setDia(9); //atribuindo o dia
  d.setAno(2018); //atribuindo o ano
  d.displayDate(); //imprimindo
  
}